### ISSUE TITLE
[Bug] 22w44c- Monitors not displaying information properly

### ISSUE THREAD

**Issue by Crimson-Artist** [tsunamayo/Starship-EVO#5001]
This has been a issue for several versions but Its worth a formal bug report. Monitors do not properly display shield and power information. For shield information, the information only appears normally on non stretched flat monitors. When linked to stretched flat monitors the information only appears on the top corner.  For wedged monitors the information just clips though the block and is not properly angled.
![20221117093030_1](https://user-images.githubusercontent.com/112352428/202505982-c6313df7-fbb2-4502-8fb9-c03e52c879f4.jpg)

For power, monitor blocks just cant be linked to it.
![20221117094019_1](https://user-images.githubusercontent.com/112352428/202506061-84e54208-c21f-4e58-8f43-10bc840097a1.jpg)

**Comment by tsunamayo** [comment]
@Crimson-Artist yes the whole system will get a refactoring. I think I will unplug the current one to be honest as it get confusing to new players. Thanks
